resources/metadata/helmExecute.yaml

```diff
@@ -63,6 +63,8 @@ spec:
           - STAGES
           - STEPS
         resourceRef:
+          - name: commonPipelineEnvironment
+            param: custom/helmRepositoryURL
```

Comment:
Not sure if a `custom` general purpose prefix is a good idea...

Reply:
this has some history in the groovy-wrapper.
Only `custom` values are also working in a compatible way for now.

In the future, when there is maybe no longer a cpe instance in groovy a refactoring could for sure make sense.